app/Http/Controllers/BackupController.php

```
@@ -19,25 +20,31 @@
 
         return Inertia::render('backups/index', [
-            'backups' => BackupResource::collection(
-                $server->backups()->with('lastFile')->simplePaginate(config('web.pagination_size'))
-            ),
+            'backups' => BackupTable::make($server->backups())->forServer($server)->simplePaginate(),
         ]);
     }
 
-    #[Get('/{backup}', name: 'backups.show')]
+    #[Get('/servers/{server}/backups/{backup}', name: 'backups.show')]
     public function show(Server $server, Backup $backup): JsonResponse
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$server'. (undefined)

(UnusedFormalParameter)